diff --git a/afirma-simple-installer/linux/instalador_deb/src/usr/share/applications/afirma.desktop b/afirma-simple-installer/linux/common/usr/share/applications/launcher.desktop.in
index 18a6b3197..fafe2fc4d 100755
--- a/afirma-simple-installer/linux/instalador_deb/src/usr/share/applications/afirma.desktop
+++ b/afirma-simple-installer/linux/common/usr/share/applications/launcher.desktop.in
@@ -1,16 +1,13 @@
 [Desktop Entry]
-Encoding=UTF-8
-Version=1.9
-Name=Autofirma
+Version=1.5
+Name=AutoFirma
 Type=Application
 Terminal=false
-Categories=Office;Utilities;Signature;Java
-Exec=/usr/bin/autofirma %u
-Icon=/usr/lib/Autofirma/Autofirma.png
+Categories=Office;Utility;Java
+Exec=autofirma %u
+Icon=@APP_ID@
 GenericName=Herramienta de firma
-Comment=Herramienta de firma
 MimeType=x-scheme-handler/afirma;
 StartupNotify=true
 StartupWMClass=autofirma
 Name[es_ES]=Autofirma
-
